### DISCUSSION TITLE
Историю расчетов выводить в обратном порядке

### DISCUSSION THREAD

**Discussion by dsamsonov1** [MarchiyGV/RotationAndDeposition#6] [Ideas]
Предлагаю вот эту историю выводить в обратном порядке:

![image](https://user-images.githubusercontent.com/61458338/179630814-c8569b83-855e-48e1-a629-4ad27d6b20df.png)

И также - она сохраняется где-то или теряется при закрытии программы?

**Comment by MarchiyGV** [comment]
- а я думал перематывать автоматически в конец списка, что будет удобнее?
- пока история не сохраняется. можно сделать запись в файл или в кэш, надо?

**Comment by dsamsonov1** [comment]
- Перематывать чревато тем, что этот скролбар будет постоянно мельтешить и бесить. Если не будет, то неважно как.

- историю желательно уметь не только сохранять, но и загружать обратно. как это правильнее реализовать питоном я не знаю. я бы делал что-то типа отображения какого-то dictionary в этот список, а сам dictionary сохранял/выгружал в файл. но это мысли динозавра, как оно там современно и молодежно, я пока не особо в курсе.

**Reply by MarchiyGV** [reply]
С точки зрения кода всё понятно, я спрашиваю про функционал: нужно ли экспортироаь эту историю куда-то или нужно только для пользования этой программой?
